scripts/auto_mesh.py

```diff
@@ -108,6 +108,17 @@ def get_connectors(hosts, port="55672", properties={}):
         c.update(properties)
     return connectors
 
+# add SASL properties, if they are set on the environment
+def add_sasl_properties(properties):
+    sasl_mechanisms = os.getenv('QDROUTERD_AUTO_MESH_SASL_MECHANISMS', 'PLAIN')
```

Comment:
I would prefer to leave the sasl mechanisms unset if not explicitly provided I think. Other than that I think it is ok.

Reply:
Ok, I changed it.